runtime/plugin/man.vim

```diff
@@ -1,6 +1,21 @@
-if get(g:, 'loaded_man', 0)
+if exists('g:loaded_man')
   finish
 endif
 let g:loaded_man = 1
 
-command! -count=0 -nargs=+ Man call man#get_page(<count>, <f-args>)
+if !exists("g:find_man_window")
+  let g:find_man_window = 1
```

Comment:
Other than the unfortunate `g:loaded_` convention, plugins must always prefix globals with the plugin name: `g:man_...`